recipes/r-extrafont/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set version = '0.17' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-extrafont
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: extrafont_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/extrafont_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/extrafont/extrafont_{{ version }}.tar.gz
+  sha256: 2f6d7d79a890424b56ddbdced361f8b9ddede5edd33e090b816b88a99315332d
+
+build:
+  number: 0
+  skip: true  # [win32]
```

Comment:
@daler can we skip this for windows. r-ttf2pt1 is not compiled for windows.
Can you give me write permissions to your fork of conda-forge maybe?

Reply:
Isn't this already being skipped? Or it should be `win` rather than `win32`? Either way, I gave you push permission to my fork so I don't slow you down ;)

Reply:
it should be win, indeed.

Reply:
I think you grated me access to conda-build, not staged-recipes :)

Reply:
Oops! Though prob not a bad thing for you to have access on that too :). I just gave you access to staged-recipes, and changed to `win`.

Reply:
Great thanks! Lets see what is still missing :)